backend/app/schemas/position.py

```
@@ -1,27 +1,360 @@
-    today_change_percent: Optional[float] = None
-    last_calculated_at: datetime
-
-    model_config = {"from_attributes": True}
+    id: int = Field(..., description="Position ID")
+    ticker: str = Field(..., description="Asset ticker symbol")
+    isin: Optional[str] = Field(None, description="ISIN identifier (XC prefixed for crypto)")
+    description: str = Field(..., description="Asset description")
+    asset_type: AssetType = Field(..., description="Asset type: stock, etf, or crypto")
+    quantity: Decimal = Field(..., description="Current quantity held")
+    average_cost: Decimal = Field(..., description="Average cost per unit including fees")
+    cost_basis: Decimal = Field(..., description="Total cost basis (quantity × average_cost)")
```

Ruff (0.13.3)

String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)